packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -525,6 +526,41 @@ abstract class AnimatedWidgetBaseState<T extends ImplicitlyAnimatedWidget> exten
 /// of [Curves.fastOutSlowIn].
 /// {@animation 250 266 https://flutter.github.io/assets-for-api-docs/assets/widgets/animated_container.mp4}
 ///
+/// {@tool snippet --template=stateful_widget_scaffold}
+///
+/// The following example transitions an AnimatedContainer between two states.
+/// It adjusts the [height], [width], [color], and [alignment] properties using
+/// [State.setState].
+///
+/// ```dart
+/// static const Curve curve = Curves.fastOutSlowIn;
+/// bool selected = false;
+///
+/// @override
+/// Widget build(BuildContext context) {
+///   return GestureDetector(
+///     onTap: () {
+///       setState(() {
+///         selected = !selected;
+///       });
+///     },
+///     child: Center(
+///       child: AnimatedContainer(
+///         width: selected ? 200.0 : 100.0,
+///         height: selected ? 100.0 : 200.0,
+///         color: selected ? Colors.red : Colors.blue,
+///         alignment:
```

Comment:
nit: I would probably put this on one line for slightly easier readability.

Reply:
I originally had it that way, but it went beyond `dartfmt`'s 80 column limit

Reply:
In Flutter you can go over 80 chars if it improves readability: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters

Reply:
No, that's only true for the code itself.  Since snippets are formatted by `dartfmt`, they have to conform to the 80 char limit.

Reply:
👍 fixed

Reply:
@gspencergoog didn't see your comment. I'll make it conform to the limit

Reply:
Interesting. I wasn't aware that there are different rules for snippets. Sorry for the confusion, @johnpryan.

Reply:
Actually, I just checked, and I'm wrong: we don't enforce 80 chars for snippets, although we do run dartfmt on them, so if you don't do 80 chars here, then the comment here won't match the formatting of the code in the docs (but nobody is likely to notice).

Reply:
Based on that, I'd suggest putting this on one line again then.